src/Helpers/ArrayDataSet.php

```diff
@@ -20,6 +21,10 @@ class ArrayDataSet
      */
     public static function renameKeys($array, $renameTo)
     {
+        if ( ! is_array($array)) {
+            return [];
```

Comment:
I’m just wondering if we shouldn’t return the original variable like a WP filter does, or maybe even convert it into an array as that's our return type. I know that anyone using this method with a non-array variable is getting an error, but to me, it feels like we should return the original input like “here’s your variable back, I couldn’t modify it.” What do you think?

Reply:
I was thinking the same, the only problem with that approach is that the function is expected to return an array. If we return something else, like null, everything after that will break.

Reply:
Yeah, then we could return it like `[$array]`

Reply:
That doesn't make any sense. You are returning this `[0=>null]` if someone passes `null`.

Reply:
Honestly, I would like to add an array as a type so that whenever someone passes null or something other than an array, it will break. But this happens for some reason, and I'm not entirely sure where this happens. So this is basically a band-aid.